docs/development/README.md

```diff
@@ -74,6 +75,29 @@ chmod +x .git/hooks/pre-commit
 _Please note that the Github workflow will still prevent this from merging
 unless the CHANGELOG is updated._
 
+### Pocket Network CLI
+
+Pocket provides a CLI for interacting with Pocket Core. The CLI is meant to be an user but also a machine friendly way for interacting with Pocket Network.
+
+The commands available are listed [here](../../app/client/cli/doc/commands/client.md).
+
+In order to build the CLI:
+
+1. Generate local files
+
+```bash
+make develop_start
+```
+
+2. Build the CLI
+
+```bash
+make build
+```
+
+The cli binary will be available at `bin/p1`.
+
+
```

Comment:
Add another point about how the available commands should be accessible via `bin/p1 --help` and the swagger UI can be opened via `make swagger-ui`

Reply:
Added a separate section about the swagger ui since although the two of them (UI and CLI) help the interaction with the RPC interface they are related to separate domains